finance_manager/src/main/resources/db/changelog/changeset/create-table-users.yaml

```diff
@@ -0,0 +1,25 @@
+databaseChangeLog:
+  - changeSet:
+      id: "2"
+      author: gs
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: u_id
+                  type: int8
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: login
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: password
+                  type: varchar(255)
+                  constraints:
+                    nullable: false

```

Comment:
Our users need some ROLE.

Reply:
I remember the alternative solution was boolean "isAdmin" or similar. It helps to avoid an additional ENUM set of roles.

Reply:
Done